packages/css/src/components/breadcrumb/breadcrumb.scss

```diff
@@ -3,6 +3,7 @@
  * Copyright Gemeente Amsterdam
```

Comment:
We should probably also add this to the following components: Button, Badge, Character Count, File List, Top Task Link.

Accordion also doesn't have this styling, but that should probably use a Heading component.

Reply:
I also did the Accordion just to be sure. Hopefully, we will remember to take it out again if we start using the Header.